std/assembly/array.ts

```diff
@@ -302,6 +302,9 @@ export class Array<T> {
   }
 
   sort(comparator: (a: T, b: T) => i32 = defaultComparator<T>()): this {
+    // TODO remove this when flow will allow trackcing null
+    assert(comparator, "The comparison function must be a function");
```

Comment:
Let's make this an `assert(comparator); // The comparison function must be a function`

Reply:
ok

Reply:
Done!